src/services/todo.services/todoServices.ts

```diff
@@ -26,12 +38,18 @@ export function createToDo(title: string, description: string): ToDo {
     id,
     createdDate,
   };
-  return toDo;
+  if (!toDo) {
```

Comment:
Where do we store a ToDo? 
This if should never trigger because the construction of the object is right above, so it will always be defined.